### PR TITLE
accessibility: issue/#2166 added role=link in place of text description

### PR DESCRIPTION
**Pull request by oliverfoster**
[#2166](https://github.com/adaptlearning/adapt_framework/issues/2166)
* Replaced text description with `role="link"`

List of outstanding PRs: https://github.com/adaptlearning/adapt_framework/issues/2206.

### CODE REVIEW

**Comment by moloko** [comment]
Have to disagree with you on that one @chucklorenz - see https://github.com/adaptlearning/adapt_framework/issues/1706

File: templates/resources.hbs [hunk 1]
```diff
@@ -26,13 +26,12 @@
...
 	<div class="resources-item drawer-item {{_type}}">
-		<button class="base resources-item-open drawer-item-open"
+		<button role="link" class="base resources-item-open drawer-item-open"
 			data-type="{{_type}}"
 			data-filename="{{filename}}"
 			data-href="{{_link}}"
 			data-force-download="{{_forceDownload}}"
-			tabindex="0"
-			aria-label="{{{title}}}. {{lookup ../model._filterButtons _type}}.{{#if description}} {{{description}}}.{{/if}} {{../model.itemAriaExternal}}.">
+			aria-label="{{lookup ../model._filterButtons _type}}. {{{title}}}. {{#if description}} {{{description}}}.{{/if}}">
```
Comment (moloko) [inline templates/resources.hbs]: this reverts the changes to the order that were made in #56

**Comment by oliverfoster** [comment]
sure

**Comment by sarveshwar-gavhane** [comment]
if itemAriaExternal no more required, would you mind removing reference from https://github.com/adaptlearning/adapt-contrib-resources/blob/master/example.json#L20 here as well, 
correct me if i am wrong.

**Pull request merged**